### ISSUE TITLE
save state load crash

### ISSUE THREAD

**Issue by bobbobbio** [bobbobbio/come_boy#2]
```
Using speedy2d renderer
Loading GamePak("POKEMON RED", MBC3+RAM+BATTERY)
Loading save state
thread '<unnamed>' panicked at 'divide by zero error when dividing duration by scalar', library\core\src\time.rs:920:31
stack backtrace:
   0: std::panicking::begin_panic_handler
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b\/library\std\src\panicking.rs:493
   1: core::panicking::panic_fmt
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b\/library\core\src\panicking.rs:92
   2: core::option::expect_failed
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b\/library\core\src\option.rs:1260
   3: core::option::Option::expect
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b\library\core\src\option.rs:349
   4: core::time::{{impl}}::div
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b\/library\core\src\time.rs:920
   5: come_boy::game_boy_emulator::Underclocker::underclock
             at C:\Users\bobbo\come_boy\src\game_boy_emulator\mod.rs:91
   6: come_boy::game_boy_emulator::GameBoyEmulator::run_inner
             at C:\Users\bobbo\come_boy\src\game_boy_emulator\mod.rs:515
   7: come_boy::game_boy_emulator::GameBoyEmulator::run
             at C:\Users\bobbo\come_boy\src\game_boy_emulator\mod.rs:531
   8: come_boy::game_boy_emulator::run_emulator<come_boy::rendering::speedy::SpeedyRenderer>
             at C:\Users\bobbo\come_boy\src\game_boy_emulator\mod.rs:686
   9: come_boy::{{impl}}::run
             at C:\Users\bobbo\come_boy\src\bin\come_boy.rs:31
  10: come_boy::bin_common::backend::speedy2d::{{impl}}::run::{{closure}}
             at C:\Users\bobbo\come_boy\src\bin_common\backend.rs:98
  11: come_boy::rendering::speedy::run_loop::{{closure}}::{{closure}}
             at C:\Users\bobbo\come_boy\src\rendering\speedy.rs:145
  12: crossbeam_utils::thread::{{impl}}::spawn::{{closure}}
             at C:\Users\bobbo\.cargo\registry\src\github.com-1ecc6299db9ec823\crossbeam-utils-0.8.3\src\thread.rs:437
  13: core::ops::function::FnOnce::call_once<closure-0,tuple<>>
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b\library\core\src\ops\function.rs:227
  14: alloc::boxed::{{impl}}::call_once
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b\library\alloc\src\boxed.rs:1328
  15: crossbeam_utils::thread::{{impl}}::spawn::{{closure}}
             at C:\Users\bobbo\.cargo\registry\src\github.com-1ecc6299db9ec823\crossbeam-utils-0.8.3\src\thread.rs:449
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by bobbobbio** [comment]
fixed in
```
commit e9a94c455031e9b18f369687199ca34badbff535 (HEAD -> work2)
Author: Remi Bernotavicius <remi@abort.cc>
Date:   Wed Mar 31 00:58:00 2021 -0700

    Set the clock speed to the default when loading a save state.
    
    fixes #2
```

**Comment by bobbobbio** [comment]
close prematurely
